docs/2_4_common_schema.adoc

```diff
@@ -1091,7 +1091,11 @@ _Note: This is reverse compared to the <<derivative>> attribute.]_
 If present, this variable is clocked.
 The value of <<clocks>> is a list of value references of <<Clock>> variables this variable depends on.
 Only variables with <<variability,`variability == discrete`>> or <<variability,`variability == tunable`>> can have this attribute.
-_[Some importers might require a variable to be dependent on a single clock for technical reasons. They could reject FMUs violating this restriction.]_
+_[Some importers might require a variable to be dependent on a single clock for technical reasons._
+_They could reject FMUs violating this restriction._ +
+_Note: It is not further restricted, which variables can be clocked to not restrict currently unknown use cases._
+_For example, an <<input>> Clock could be a clocked variable of another (<<input>> or <<output>>) clock to indicate that it can only be activated when that Clock is active.]_
```

Comment:
@andreas-junghanns @masoud-najafi this contradicts "[Output Clocks are intended to enable an FMU to provide a trigger to the outside world, e.g. to a triggered input Clock of another FMU. Note, that if intended to activate model partitions or a Clock of the same FMU calling fmi3GetInterval for a countdown Clock and evaluating the qualifiers is recommended. An example is provided in Section 5.3.]". I would rather not have output clocks trigger input clocks of the same FMU but recommend to use countdown clocks instead.

Reply:
... and I think we all agree with your preference. The question here is: Should we restrict the standard to mandate our preference, or should we allow such "wired" usages? In our discussion we tend to follow the Klausian camp: "Why restrict a usage simply because we do not know what it is good for."

Reply:
@andreas-junghanns ... No we're not discussing to restrict it. It is a recommendation. To be consistent within the standard we should not give an example that ignores the recommendation. That's all. Can we please adjust this?

Reply:
Suppose that an input clock c1 that depends on another input/output clock c2, then is c2 sub-samples c1, or it indicates dependency of c2 on c1? It is not clear for me....

Reply:
> @andreas-junghanns ... No we're not discussing to restrict it. It is a recommendation. To be consistent within the standard we should not give an example that ignores the recommendation. That's all. Can we please adjust this?

@MBlesken : I don´t understand: adjust what? The example? The text?

Reply:
> Suppose that an input clock c1 that depends on another input/output clock c2, then is c2 sub-samples c1, or it indicates dependency of c2 on c1? It is not clear for me....

If c1 depends on c2, how can c2 be a sub-sample of c1? I do not understand your question?